ERCS/erc-7654.md

```diff
@@ -0,0 +1,146 @@
+---
+eip: 7654
+title: Request Method Types
+description: Use a set of request methods to indicate the type of action to take on the contract.
+author: Rickey (@HelloRickey)
+discussions-to: https://ethereum-magicians.org/t/erc-7654-request-method-types/19183
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-03-13
+---
+
+## Abstract  
+
+Usually each abi file corresponds to a specific contract, and the client cannot use the same abi to call different functions of different contracts. Contract Request Methods redefines the request method of the contract, so that multiple contracts can be called without using multiple different abi files.
```

Comment:
Now you've completely removed any mention of `GET`/`POST`/etc. from your abstract. If you're proposal is centred around these verbs, they probably deserve a mention here.

For context (from [EIP-1](https://eips.ethereum.org/EIPS/eip-1)):

> Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.